recipes/ssht/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
we can probably remove the entire `test_v1_package` folder - if it did not exist prior to this PR, we are no longer require newly updated recipes to include it